dotcom-rendering/makefile

```diff
@@ -62,7 +62,7 @@ start: install
 start-prod:
 	$(call log, "starting PROD server...")
 	@echo '' # just a spacer
-	NODE_ENV=production /usr/local/node/pm2 start --uid dotcom-rendering --gid frontend dist/server.js
+	NODE_ENV=production /usr/local/node/pm2 start --uid dotcom-rendering --gid server dist/server.js
```

Comment:
I've renamed the pm2 `gid` here also. I'll assume nothing is expecting `frontend` (logs, analytics etc) but I'll connect to a CODE instance to sanity check.

Reply:
`gid` is the user's group not the pm2 id for the process as I had assumed. Changing this preventing the app from starting up so I've reverted to `--gid frontend`.